docs/examples-profiling.md

```diff
@@ -0,0 +1,101 @@
+# Profiling WebAssembly
```

Comment:
This is great.  I see the link to it in SUMMAR.mdY, but does the SUMMARY.md get published anywhere? I could only find it on the github view of this branch by clicking through the source tree and not any official links to docs found on the opening Readme.

Reply:
Ah yeah these are auto-published from the master branch at https://bytecodealliance.github.io/wasmtime/

Reply:
In particular the `SUMMARY.md` is the table of contents on the left hand side. More info in the `mdbook` user guide: https://rust-lang.github.io/mdBook/format/summary.html

Reply:
@fitzgen .. Ahh I see. Thanks. I clicked on docs here linked to at the bottom here: https://github.com/alexcrichton/wasmtime/tree/profiling for this pr, but I didn't notice that link was going to be linking back to master. Got to be carefully following links like that.